commands/upgrade/util.py

```diff
@@ -158,6 +158,9 @@ def handle_output_level(args):
         os.environ['LEAPP_VERBOSE'] = os.getenv('LEAPP_VERBOSE', '0')
 
 
+# NOTE(ivasilev) Please make sure you are not calling prepare_configuration after first reboot.
```

Comment:
@Rezney addressed your comment and also added a note to (hopefully) not shoot ourselves in the foot next time.
But it was cool :)